src/templates/components/qaSection/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { QaSection as FormattedQaSection } from '@/types/formatted/qaSection'
+import { QaCollapsiblePanel } from '../qaCollapsiblePanel'
+import { QaParagraph } from '../qaParagraph'
+import { QaParagraph as FormattedQaParagraph } from '@/types/formatted/qaParagraph'
+import { QaGroup as FormattedQaGroup } from '@/types/formatted/qaGroup'
+
+export function QaSection({
+  header,
+  intro,
+  questions,
+  displayAccordion,
+}: FormattedQaSection | FormattedQaGroup) {
+  const setHeaderh3 = header ? true : false
+  return (
+    <div data-template="paragraphs/q_a_section">
+      {header && <h2>{header}</h2>}
+      {intro && <p>{intro}</p>}
+      {displayAccordion ? (
+        <QaCollapsiblePanel questions={questions} />
+      ) : (
+        questions.map((questionObject: FormattedQaParagraph, index) => (
```

Comment:
This feels interesting to me. Despite TypeScript not yelling at this, it feels like `FormattedQaParagraph` can't work here because `questions` can actually be `FormattedParagraph[]` or `QaGroupQa[]`. Since `FormattedParagraph` encompasses a number of different types, they won't all map to `FormattedQaParagraph`.

It seems to me that we need to type `questions` differently. I'm leaving some other comments in some other places to this end.

Reply:
It will need to be formattedParagraph to accommodate the change below to <Paragraph>